web/src/components/tasks/TaxInput.tsx

```diff
@@ -106,6 +106,23 @@ export const TaxInput = (props: Props): ReactElement => {
     </div>
   );
 
+  const showNeedsAccountModalForGuest = (): (() => void) | undefined => {
+    if (isAuthenticated === IsAuthenticated.FALSE) {
+      return () => {
+        return setShowNeedsAccountModal(true);
+      };
+    }
+    return undefined;
```

Comment:
[dust] This is about as nit picky as I could be, but you don't need to explicitly return `undefined`. You can just `return`.